integration-tests/__tests__/each.test.js

```diff
@@ -14,10 +14,8 @@ const runJest = require('../runJest');
 const {extractSummary} = require('../Utils');
 const dir = path.resolve(__dirname, '../each');
 const SkipOnWindows = require('../../scripts/SkipOnWindows');
-const SkipOnJestCircus = require('../../scripts/SkipOnJestCircus');
 
 SkipOnWindows.suite();
```

Comment:
btw, do we need skipping it on windows?

Reply:
I think we tried removing this before and it failed 🤷‍♂️

Reply:
See https://github.com/facebook/jest/pull/6102#issuecomment-385711772

Reply:
we should probably add a helper fixing the differing symbols on windows and unix so that the integration tests can run. separate issue, though